src/proto/single/mutation.rs

```diff
@@ -0,0 +1,112 @@
+use crate::JobId;
+use derive_builder::Builder;
+
+#[cfg(doc)]
+use crate::{Client, Job};
+
+/// Mutation target set.
+#[derive(Clone, Debug, Default, PartialEq, Eq, Serialize)]
+#[serde(rename_all = "lowercase")]
+#[non_exhaustive]
+pub enum MutationTarget {
```

Comment:
Is this actually specific to mutations? Feels like a general purpose job set specifier to me.

It's also weird to me that the "mutation target" isn't just another filter, given that seems to be the function it has.